### PR TITLE
Remove css to add bottom spacing to sub menu

### LINKED ISSUE #288: Sub-menu: Add bottom padding/spacing
**Describe the bug**
The last item in the sub-menu, or the sub-menu list itself, needs some more spacing at the bottom:
See screenshot.

the sub-menu is missing space after the last menu item

### PR DESCRIPTION
**Pull request by abhijitrakas**
image

WP.org Username: [abhijitrakas](https://profiles.wordpress.org/abhijitrakas/)

Closes: https://github.com/WordPress/twentytwentyone/issues/288

### CODE REVIEW

File: assets/sass/06-components/navigation.scss [hunk 1]
```diff
@@ -351,10 +351,6 @@
 		.menu-item {
 			margin: calc(2 * var(--primary-nav--padding)) 0;
 
-			&:last-child() {
```
Comment (carolinan) [inline assets/sass/06-components/navigation.scss:354]: Removing this affects both menus, so there is now a space below the sub menus in the responsive version of the menu.

Comment (kishanjasani) [reply assets/sass/06-components/navigation.scss:354]: Simply add media query for this section something like `@include media(mobile) {}`.

Comment (carolinan) [reply assets/sass/06-components/navigation.scss:354]: (mobile is for mobile and and up, mobile-only is for mobile only)

Comment (abhijitrakas) [reply assets/sass/06-components/navigation.scss:354]: Thank you, folks. I am done with the changes.

**Review by kishanjasani — APPROVED** [review]
LGTM 👍

**Pull request merged**